cpp/installer.cc

```diff
@@ -813,6 +814,36 @@ void InstallFullTextBackendCronjobs() {
 }
 
 
+void WaitForElasticsearchReady() {
+     const std::string host("localhost");
+     const std::string base_url("http://" + host + ":9200/");
+     const unsigned MAX_ITERATIONS(5);
+     const unsigned SLEEP_TIME(5 * 1000 * 1000);
```

Comment:
Please always add either a suffix in the variable or constant name indicating the unit or a short trailing comment like " // ms".